website/docs/r/sslcert.html.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: "gridscale"
+page_title: "gridscale: gridscale_ssl_certificate"
+sidebar_current: "docs-gridscale-resource-ssl_certificate"
+description: |-
+  Manages a SSL Certificate resource in gridscale.
+---
+
+# gridscale_ssl_certificate
+
+Provides a SSL Certificate resource. This can be used to create and delete SSL Certificates.
```

Comment:
Change: `"This can be used to create and delete SSL Certificates"` to 

`"This can be used to create and delete TLS/SSL certificates"`